### PR TITLE
Change border radius of calendar

### PR DESCRIPTION
**Pull request by samiede**
I honestly don't remember the changes I made because it's been so long ago, but it was only a few lines of code.

### CODE REVIEW

**Comment by chenei** [comment]
Can you please explain more what it does?
Why we can’t we change the border radius today using theme prop?

**Comment by samiede** [comment]
I'm not sure, at the time I wrote this small PR, it was not possible to change the radius via any customisation as it was hardcoded

File: src/calendar-list/index.js [hunk 1]
```diff
@@ -59,22 +61,23 @@ class CalendarList extends Component {
     showScrollIndicator: false,
     scrollEnabled: true,
     scrollsToTop: false,
-    removeClippedSubviews: Platform.OS === 'android' ? false : true
+    removeClippedSubviews: Platform.OS === 'android' ? false : true,
+    scrollToDate: true
```
Comment (Inbal-Tish) [inline src/calendar-list/index.js:65]: What is 'scrollToDate' prop has to do with fixing borderRadius?

File: src/calendar/day/period/index.js [hunk 2]
```diff
@@ -130,7 +130,7 @@ class Day extends Component {
 
     if (this.props.marking) {
       containerStyle.push({
-        borderRadius: 17
+        borderRadius: this.style.base.borderRadius
```
Comment (Inbal-Tish) [inline src/calendar/day/period/index.js:133]: This is the actual fix of this PR, right? Where is the 'style.base.borderRadius' is coming from? Why you're not using the 'theme' object and 'appStyle' to pass styles to the component?

**Comment by emilisb** [comment]
Hi @samiede,

Any plans to finish with this?

File: src/calendar-list/index.js (hunk 1 above)
Comment (samiede) [reply src/calendar-list/index.js:65]: I think this was a different issue where the calendar would scroll to the date automatically, even if you don't necessarily want it to.

File: src/calendar/day/period/index.js (hunk 2 above)
Comment (samiede) [reply src/calendar/day/period/index.js:133]: If I remember correctly, this is what I wanted to do, but the border radius was hardcoded at the time, i.e. this part of the styling could not be overwritten by the appStyle object.
I apologise of this is obsolete by now, but I don't have access to the companies code anymore, so I'm not 100% sure in what situation I needed this.

**Comment by samiede** [comment]
Hey @emilisb, I haven't worked on this since April, so I'm not quite sure anymore, but I have commented under the remarks with everything I remember!

**Pull request closed without merging**